package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@anrok/mammoth",
   "license": "MIT",
-  "version": "0.3.2",
+  "version": "0.4.0",
```

Comment:
Technically this should be version 1.0 since this is a breaking change, but I'm not sure if we consider this officially released yet.